hw/ip/i2c/data/i2c.hjson

```diff
@@ -524,11 +565,11 @@
       swaccess: "rw"
       hwaccess: "hro"
       fields: [
-        { bits: "15:0"
+        { bits: "9:0"
```

Comment:
With a maximum core clock of 1 GHz, the rise and fall times are only 10-bit counters to accommodate a maximum of 1000 ns (and really, the fall time could be reduced further, since the max allowed by spec is only 300 ns).

Reply:
> So we can reduce the width of `T_F` by another bit without loss, no?

We can. I simply got lazy and matched rise and fall widths.

Reply:
Done. I also removed some bits from TSU_DAT, which had 4 more bits than necessary. THD_DAT likely also has more bits than necessary, but there's a line about CBUS-compatible controllers needing 5 us, so I left it alone for now.